prisma/migrations/20230103180746_multi_asset_rename/migration.sql

```diff
@@ -0,0 +1,97 @@
+  - You are about to drop the column `masp_transaction_id` on the `events` table. All the data in the column will be lost.
+  - You are about to drop the column `masp_burn_count` on the `user_points` table. All the data in the column will be lost.
+  - You are about to drop the column `masp_burn_last_occurred_at` on the `user_points` table. All the data in the column will be lost.
+  - You are about to drop the column `masp_burn_points` on the `user_points` table. All the data in the column will be lost.
+  - You are about to drop the column `masp_mint_count` on the `user_points` table. All the data in the column will be lost.
+  - You are about to drop the column `masp_mint_last_occurred_at` on the `user_points` table. All the data in the column will be lost.
+  - You are about to drop the column `masp_mint_points` on the `user_points` table. All the data in the column will be lost.
+  - You are about to drop the column `masp_transfer_count` on the `user_points` table. All the data in the column will be lost.
+  - You are about to drop the column `masp_transfer_last_occurred_at` on the `user_points` table. All the data in the column will be lost.
+  - You are about to drop the column `masp_transfer_points` on the `user_points` table. All the data in the column will be lost.
+  - You are about to drop the `masp_transaction_head` table. If the table is not empty, all the data it contains will be lost.
+  - You are about to drop the `masp_transactions` table. If the table is not empty, all the data it contains will be lost.
+  - A unique constraint covering the columns `[multi_asset_id]` on the table `events` will be added. If there are existing duplicate values, this will fail.
+
+*/
+-- AlterTable
+ALTER TABLE "events" DROP COLUMN "masp_transaction_id",
+ADD COLUMN     "multi_asset_id" INTEGER;
+
+-- AlterTable
+ALTER TABLE "user_points" DROP COLUMN "masp_burn_count",
+DROP COLUMN "masp_burn_last_occurred_at",
+DROP COLUMN "masp_burn_points",
+DROP COLUMN "masp_mint_count",
+DROP COLUMN "masp_mint_last_occurred_at",
+DROP COLUMN "masp_mint_points",
+DROP COLUMN "masp_transfer_count",
+DROP COLUMN "masp_transfer_last_occurred_at",
+DROP COLUMN "masp_transfer_points",
+ADD COLUMN     "multi_asset_burn_count" INTEGER NOT NULL DEFAULT 0,
+ADD COLUMN     "multi_asset_burn_last_occurred_at" TIMESTAMP(6),
+ADD COLUMN     "multi_asset_burn_points" INTEGER NOT NULL DEFAULT 0,
+ADD COLUMN     "multi_asset_mint_count" INTEGER NOT NULL DEFAULT 0,
+ADD COLUMN     "multi_asset_mint_last_occurred_at" TIMESTAMP(6),
+ADD COLUMN     "multi_asset_mint_points" INTEGER NOT NULL DEFAULT 0,
+ADD COLUMN     "multi_asset_transfer_count" INTEGER NOT NULL DEFAULT 0,
+ADD COLUMN     "multi_asset_transfer_last_occurred_at" TIMESTAMP(6),
+ADD COLUMN     "multi_asset_transfer_points" INTEGER NOT NULL DEFAULT 0;
+
+-- DropTable
+DROP TABLE "masp_transaction_head";
+
+-- DropTable
```

Comment:
Had to add `CASCADE` here to get drop to work, needed to delete the associated unique index